pkg/goalesce/utils.go

```diff
@@ -0,0 +1,66 @@
+package goalesceutils
+
+import (
+	"github.com/adutra/goalesce"
```

Comment:
ahem... can we fork the goalesce project into the k8ssandra org and reference that version instead?

Reply:
As we discussed offline, this is a project that I started outside K8ssandra, and I intend to maintain it. Of course you could fork it if you want to, but at this point, I don't see any valid reason to do so.

Reply:
sounds good 👍